requirements-dev.txt

```diff
@@ -50,6 +50,7 @@ httpx==0.25.0; python_version >= '3.8'
 idna==3.4; python_version >= '3.5'
 orjson==3.9.7; python_version >= '3.7'
 psutil==5.9.5; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+py-cpuinfo==9.0.0
```

Comment:
dev?

Reply:
seems like dev is a superset of regular deps